SQL/Cassandra.sublime-syntax

```diff
@@ -0,0 +1,238 @@
+%YAML 1.2
+---
+# https://cassandra.apache.org/doc/latest/cassandra/cql
+name: Cassandra Query Language
```

Comment:
It would indeed make it easier to know what's changed and needs support added for but I haven't seen any easy way to correlate which version of Apache Cassandra ships with which version of Cassandra Query Language support. As the aim is always to support the latest version and we may not have covered all features yet, I vote to leave any comments about versioning out for now.